assets/app/features/upgrade/controllers/upgrade7-upgrade-admin.controller.spec.js

```diff
@@ -0,0 +1,161 @@
+/*global bard $controller should $httpBackend upgradeUpgradeAdminFactory assert $q $rootScope */
+describe('Upgrade Flow - Upgrade Admin Server Controller', function () {
+    var controller,
+/*        completedUpgradeResponse = {
+            completed: true
+        },*/
+        incompleteUpgradeResponse = {
+            completed: false
+        },
+        errorList = ['1', '2', '3'],
+        errorResponse = {
+            data: { errors: errorList }
+        };
+
+    beforeEach(function() {
+        //Setup the module and dependencies to be used.
+        bard.appModule('crowbarApp');
+        bard.inject('$controller', '$q', '$httpBackend', '$rootScope', 'upgradeUpgradeAdminFactory');
+
+        //Create the controller
+        controller = $controller('Upgrade7UpgradeAdminController');
+
+        //Mock requests that are expected to be made
+        $httpBackend.expectGET('app/features/upgrade/i18n/en.json').respond({});
+        $httpBackend.flush();
+    });
+
+    // Verify no unexpected http call has been made
+    bard.verifyNoOutstandingHttpRequests();
+
+    it('should exist', function () {
+        should.exist(controller);
+    });
+
+    describe('nextStep function', function () {
```

Comment:
I think this section could be removed, since `nextStep` function is part of (or should be soon) `https://github.com/crowbar/crowbar-angular/blob/master/assets/app/features/upgrade/upgrade7.controller.js`